Dockerfile

```diff
@@ -12,21 +12,14 @@ RUN \
 
 ENV PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
 
-RUN pip install pipenv
-
-# -----------------------------------------
-COPY Pipfile /tmp/Pipfile
-COPY Pipfile.lock /tmp/Pipfile.lock
-RUN cd /tmp && pipenv install --system --deploy --ignore-pipfile --dev
-
 COPY ./ /kb/module
-RUN mkdir -p /kb/module/work
-RUN chmod -R a+rw /kb/module
-
 WORKDIR /kb/module
 
-# really need a test build and a prod build. Not sure that's possible via sdk.
-# RUN pipenv install --system --deploy --ignore-pipfile --dev
+RUN mkdir -p /kb/module/work \
```

Comment:
How true; update in place.

I'd also like to, not now, but sometime, replace the dockerize  with the semi-official fork https://github.com/powerman/dockerize. Steve had forked it to add support for KBase, but most of that has been merged into this fork, as well as other fixes, improvements - and it is supported.
The one-time downside of that switch is that there is a small difference in the command line options which can affect rancher setups, but in this case the dockerize is buried in the entrypoint file, so that is not an issue.